examples/tests/sdk_tests/std/duration.w

```diff
@@ -2,10 +2,12 @@
 // seconds
 
 assert(12m.seconds == 12 * 60);
+assert(duration.fromMinutes(12).seconds == 12 * 60);
 
-// TODO: https://github.com/winglang/wing/issues/1629
-// test "d" {
-//   let d = 12m;
-// }
+test "duration" {
+  // TODO: https://github.com/winglang/wing/issues/1629
+  // assert(12m.seconds == 12 * 60);
```

Comment:
This is still unresolved?

Reply:
@eladb - see #2902 - can you please give it a review?